Toolchain/default.nix

```diff
@@ -3,42 +3,61 @@
 }:
 with pkgs;
 
-mkShell.override { stdenv = gcc13Stdenv; } {
-  packages = [
-    ccache
-    cmake
-    curl
-    e2fsprogs
-    fuse2fs
-    gcc13
-    gmp
-    libmpc
-    mpfr
-    ninja
-    patch
-    pkg-config
-    rsync
-    texinfo
-    unzip
-    # To build the GRUB disk image
-    grub2
-    libxcrypt
-    openssl
-    parted
-    qemu
-    python3
-    # For building and installing ports
-    autoconf
-    automake
-    gperf
-    imagemagick
-    libtool
-    # For clangd and clang-format
-    clang-tools
-    # For LibWeb-related formatting
-    nodePackages.prettier
-    # For the pre-commit hooks
-    pre-commit
+mkShell.override { stdenv = gccStdenv; } {
+  packages =
+    [
+      git
+      flex
```

Comment:
:redlinus:, list no longer attempts to be sorted (within one section). And now I also noticed that the last four packages in the section also aren't sorted.